e2e/netman/manager.go

```diff
@@ -353,6 +359,19 @@ func (m *manager) deployIfNeeded(ctx context.Context, chain types.EVMChain, back
 	return addr, receipt.BlockNumber.Uint64(), nil
 }
 
+// destChainIDs returns all configured destination chain ids for a given source.
+func (m *manager) destChainIDs(srcChainID uint64) []uint64 {
+	var destChainIDs []uint64
+
+	for id := range m.Portals() {
+		if id != srcChainID {
+			destChainIDs = append(destChainIDs, id)
+		}
+	}
```

Comment:
Is there a case where we don't want some chain to have another chain as destination apart from `srcID == destID`? Is it possible that this is required in future?

Reply:
Perhaps. But I can't think of a reason why rn